tutorials/setup.md

```diff
@@ -11,7 +11,41 @@ Please note: We only support Mac and Windows computers.
 they figure things out by themselves or are is Linux not allowed?
 
 
-## Setting Up the Git Repository
+## Install Python
```

Comment:
Moved Python installation first because (with Lisa's tutorial) it also has students install Homebrew or Chocolatey, which they can use to install git.